ostelco-ios-client/Assets/Assets.xcassets/icIcon.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "idIcon.pdf"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

Comment:
no new line at EOF

Reply:
Is this something automatically created ?

Reply:
yes, this is auto-created by xcode, so I can't add the newline to it unfortunately. That red thing makes me bonkers as well